sdks/python/apache_beam/runners/interactive/display/pcoll_visualization_test.py

```diff
@@ -0,0 +1,156 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Tests for apache_beam.runners.interactive.display.pcoll_visualization."""
+from __future__ import absolute_import
+
+import sys
+import time
+import unittest
+
+import apache_beam as beam
+from apache_beam.runners import runner
+from apache_beam.runners.interactive import interactive_environment as ie
+from apache_beam.runners.interactive.display import pcoll_visualization as pv
+
+# Work around nose tests using Python2 without unittest.mock module.
```

Comment:
Could you add a py2 todo here, for this to be removed later. You might be able to use BEAM-1251 or maybe @tvalentyn can suggest a better JIRA.

Reply:
I think this part is OK. We will later have to search the codebase for all comments including Python2, Py2, Python 3, etc for this cleanup. If you want a JIRA, feel free to use BEAM-7372.

Reply:
Thanks @aaltay and @tvalentyn, I also have a [JIRA](https://issues.apache.org/jira/browse/BEAM-8288) to clean up for interactive packages. I'll use that one for this too.